### DISCUSSION TITLE
How to get the content of an iframe?

### DISCUSSION THREAD

**Discussion by thalesfsp** [apify/crawlee-python#517] [Q&A] [answered]
Thank you!

**Comment by janbuchar** [comment]
Hello, in general you can read its `src` attribute and make a separate request to that. Which crawler are you using?

**Reply by thalesfsp** [reply]
Hello @janbuchar `PlaywrightCrawler`. Would you guys have or provide an example on how to achieve that? Tax!

**Reply by janbuchar** [reply, marked as answer]
Then I'd advise you to go through https://playwright.dev/python/docs/api/class-frame.
